docs/getting-started/integrations-cdn/angular.md

```diff
@@ -101,36 +101,39 @@ This guide assumes you already have a Angular project. To create such a project,
 
 ### Installing the Angular component from npm
 
-First, install the official shared utils for CKEditorr&nbsp;5 integrations:
+First, install the [CKEditor&nbsp;5 WYSIWYG editor component for Angular](https://www.npmjs.com/package/@ckeditor/ckeditor5-angular):
 
 ```bash
-npm install @ckeditor/ckeditor5-integrations-common
+npm install @ckeditor/ckeditor5-angular
 ```
 
 ### Using the Angular component
 
-Then, to use CKEditor&nbsp;5 with CDN, you need to import the `loadCKEditorCloud` function and call it inside `ngOnInit` with the `version` provided in the configuration.
+To use CKEditor&nbsp;5 with CDN, you need to import the `loadCKEditorCloud` function. Then, call it inside the `ngOnInit` lifecycle hook with the `version` provided in the configuration.
 
 ```ts
-import { Component, loadCKEditorCloud, CKEditorCloudResult } from '@angular/core';
+// app.component.ts
+
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CKEditorModule, loadCKEditorCloud, CKEditorCloudResult } from '@ckeditor/ckeditor5-angular';
 import { ClassicEditor, EditorConfig } from 'https://cdn.ckeditor.com/typings/ckeditor5.d.ts';
 
 @Component( {
 	selector: 'app-root',
 	templateUrl: './app.component.html',
 	styleUrls: ['./app.component.css'],
+	imports: [ CommonModule, CKEditorModule ],
+	standalone: true
 } )
-export class SimpleCdnUsageComponent {
-	public Editor: typeof ClassicEditor | null = null;
+export class AppComponent {
+	public Editor: typeof ClassicEditor | undefined = undefined;
```

Comment:
it may work with `typeof ClassicEditor | null` imo

Reply:
Fixed ✅